models/migrations/v88.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"crypto/md5"
+	"fmt"
+
+	"code.gitea.io/gitea/modules/util"
+
+	"github.com/go-xorm/xorm"
+)
+
+func hashContext(context string) string {
+	return fmt.Sprintf("%x", md5.Sum([]byte(context)))
+}
+
+func addCommitStatusContext(x *xorm.Engine) error {
```

Comment:
I don't think this migration is run, it hasn't been added to `gitea/models/migrations/migrations.go`

Reply:
Good catch, fixed.